src/main/java/org/mongounit/MongoUnitUtil.java

```diff
@@ -176,12 +176,11 @@ public static void toDatabase(
   public static void dropAllCollectionsInDatabase(MongoDatabase mongoDatabase) {
 
     // Iterate over all collections in the db and drop them
-    for (String collectionName : mongoDatabase.listCollectionNames()) {
-
+    mongoDatabase.listCollectionNames().forEach(collectionName -> {
```

Comment:
@alexisGandar This update is just to use the newer syntax in Java, correct? I don't think this code was breaking with the version update, was it?

Reply:
It's not related to the java version. It's related to the upgrade of MongoDB drivers. This is the reason of this PR. After upgrading spring boot to 3.3.0 the MongoDB driver sub-dependency were also updated and in the update they changed the return type of the listCollectionNames. This change was required to avoid an error.